From 92fbc9551dc98e4bd83b5871cd23e88c4c2c4816 Mon Sep 17 00:00:00 2001
From: Jake Weinstein <jake@aospa.co>
Date: Mon, 7 Aug 2023 14:51:00 +0900
Subject: [PATCH] Use -march=armv9-a+nosve for Armv9-A arch variant

SVE is not supported by current compilers, but
other Armv9-A features can be enabled.

Test: llvm-objdump -d
Change-Id: I99db407eb96d82d189378fd5abd4cc80ab802f20
---
 cc/config/arm64_device.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cc/config/arm64_device.go b/cc/config/arm64_device.go
index 31781c7fc..029fd95b4 100644
--- a/cc/config/arm64_device.go
+++ b/cc/config/arm64_device.go
@@ -44,7 +44,7 @@ var (
 		// On ARMv9 and later, Pointer Authentication Codes (PAC) are mandatory,
 		// so -fstack-protector is unnecessary.
 		"armv9-a": []string{
-			"-march=armv9-a",
+			"-march=armv9-a+nosve",
 			"-mbranch-protection=standard",
 			"-fno-stack-protector",
 		},
-- 
2.47.1.windows.1

